test/testData/mockReportData.json

```diff
@@ -251,7 +260,119 @@
         "kbCategory": "test kb category",
         "msiScore": 0.0
     },
+    "kbMatchedStatements": [
+        {
+            "approvedTherapy": false,
+            "category": "therapeutic",
+            "context": "test multivariant statement",
+            "disease": "colorectal cancer [DOID:9256]",
+            "kbStatementId": "#999:999",
+            "matchedCancer": true,
+            "reference": "pmid:TEST1",
+            "relevance": "resistance",
+            "iprEvidenceLevel": "IPR-D",
+            "externalSource": "IPRKB",
+            "reviewStatus": "pending",
+            "kbData": {},
+            "kbMatches": [
+                {
+                    "kbVariant": "APC:p.T1556fs",
+                    "variant": "a12458ff7c8eb865656fd61f670cd0d5",
+                    "variantType": "mut",
+                    "kbVariantId": "#158:1343"
+                },
+                {
+                    "kbVariant": "ZFP36L2:p.Q401del",
+                    "variant": "TEST1",
+                    "variantType": "mut",
+                    "kbVariantId": "#999:9999"
+                }
+            ]
+        }
+    ],
     "kbMatches": [
```

Comment:
My understanding is that each kbMatches record is a 1-1-1 relationship between the observed variant (variant & variantType), a gkb statement (kbStatementId), and the variant it match to in that statement's conditions (kbVariantId). So 1 record for each of these combinations, right?

Reply:
At the moment it's possible to do a 1-many relationship with one kbmatch to many kbstatements, in this section. That functionality is not removed in this pr, it just adds the ability to load in the inverse way (1-many with one kbstatement to many kbmatches).

Reply:
I meant for a sigle kbMatches record. Are you talking about the capability to create multiple records using the same observed variant but linked to different kbStatementId?